subsys/net/lib/nrf_provisioning/src/nrf_provisioning.c

```diff
@@ -574,8 +577,9 @@ int nrf_provisioning_req(void)
 				LOG_DBG("Saving the latest command id");
 				commit_latest_cmd_id();
 			}
-			dm.cb(dm.user_data);
+			dm.cb(NRF_PROVISIONING_EVENT_STOP, dm.user_data);
 		}
+		dm.cb(NRF_PROVISIONING_EVENT_DONE, dm.user_data);
```

Comment:
This looks like now that the `STOP` event is only coming if the connection was succesful.

I think the START-STOP should come in pairs.. They must be indicated regardless of the outcome.

The `DONE` should only be indicated if there was new data.